Dockerfile

```diff
@@ -129,7 +129,7 @@ RUN set -x \
 # IMPORTANT: If the version of Go is updated, the Windows to Linux CI machines
 #            will need updating, to avoid errors. Ping #docker-maintainers on IRC
 #            with a heads-up.
-ENV GO_VERSION 1.7.3
+ENV GO_VERSION 1.8beta1
```

Comment:
Should we change all occurrences? (I remember there's a lot more places where it's defined)